src/components/selectVersionElement.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+
+// TODO: I doubt this needs an ID.
+const SelectVersionElement = props => {
```

Comment:
Alternatively `const SelectVersionElement = props => (` and no `return` statement.